files/tasks.py

```
@@ -1246,48 +1273,119 @@ def beat_test(x, y):
-            # Kill the process
-            subprocess.run(["kill", "-9", pid], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            return result
+        output = result.stdout.decode("utf-8").strip()
+        if not output:
+            return None
+        pids = output.split("\n")
+        for pid in pids:
+            pid = pid.strip()
+            if pid:
+                try:
+                    subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)